package.json

```diff
@@ -1,5 +1,5 @@
 {
   "dependencies": {
-    "couchbase": "github:couchbase/couchnode#master"
+    "couchbase": "git+https://github.com/couchbase/couchnode.git#master"
```

Comment:
Just been having a look at this btw as I'm trying to setup automation, I don't think this will actually work. 
You *need* to have a `package.json` in each "module" unfortunately. Though I have figured out a way to run npm install once and ensure it looks at all the other package.json's to build the dependencies needed.

Note that this assumes your `root` package.json is in the `modules` directory:
```
{
	"name": "docs-nodejs-examples",
	"dependencies": {
		"devguide": "file:devguide/examples/nodejs",
		"hello-world": "file:hello-world/examples",
		"howtos": "file:howtos/examples"
	}
}
```

I'm also setting up a utility script that will help update the Couchbase version in each package.json so you don't have to do it manually. In fact, I noticed alot of them still say 3.1 (so they need to be updated to 3.2)

This is what I found when investigating this:
https://stackoverflow.com/questions/24478799/notating-multiple-modules-in-the-same-package-json-file

Let me know if I'm not making sense, 😄  need some coffee...

Reply:
Hmm, I'm not 100% sure why I made that change, probably for local testing at some point.
I'll revert, thanks.